src/App.js

```diff
@@ -1,10 +1,41 @@
 import './App.css';
 import Menu from "./component/menu/menu"
+import Sidebar from "./component/sidebar/Sidebar"
+import MovieContant from "./component/movie/MovieContant"
+import Footer from "./component/footer/Footer"
+import {GenresProvider} from "./component/sidebar/fillter/context/GenresContext";
+import {LanguageProvider} from "./component/sidebar/fillter/context/languageContext";
+import {PageProvider} from "./component/sidebar/fillter/context/PageContext";
+import {KeywordsProvider} from "./component/sidebar/fillter/context/KeywordContext";
+import { BrowserRouter } from 'react-router-dom';
+
+
 function App() {
   return (
+    <KeywordsProvider>
+    <PageProvider>
+<GenresProvider>
+<LanguageProvider>
+<BrowserRouter>
+
+
```

Comment:
Too many context providers, You only need one context provider containing one state value that contains all filters.
```
<FiltersProvider>
//rest of the code
</ FiltersProvider>
```